include/openenclave/internal/syscall/dirent.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
For files that were moved to the new header location, did you do it via `git mv`? If possible, it would be nice to preserve the lineage of the file history instead of having deletes and adds.

Reply:
I'm sorry but this is a git glitch.